saleor/graphql/attribute/filters.py

```diff
@@ -58,6 +58,16 @@ def filter_attribute_type(qs, _, value):
     return qs.filter(type=value)
 
 
+class AttributeValueFilter(django_filters.FilterSet):
+    search = django_filters.CharFilter(
+        method=filter_fields_containing_value("slug", "name")
```

Comment:
Currently, I working on dropping `filter_by_query_param` and `filter_fields_containing_value`. Can you write a custom function for this search?

Reply:
Should we search through `value`?

Reply:
I'll check where we use this field `value` - for most attributes it's empty, and I think searching by name and slug should be enough. Need to confirm.

Reply:
Confirmed - we use `value` only for storing RGB values in case of swatch attributes.